File: docs/source/_src_snippets/simulation/environment_setup/environment_models/spherical_harmonics_gravity.py
gravitational_parameter = ...
reference_radius = ...

normalized_cosine_coefficients = ... # NOTE: entry (i,j) denotes coefficient at degree i and order j
normalized_sine_coefficients = ... # NOTE: entry (i,j) denotes coefficient at degree i and order j

associated_reference_frame = ... # NOTE: provide as string

body_settings.get_body( "Earth" ).gravity_field_settings = environment_setup.gravity_field.spherical_harmonic( gravitational_parameter, 
	reference_radius, normalized_cosine_coefficients, normalized_sine_coefficients, associated_reference_frame )

File: docs/source/_snippets/simulation/environment_setup/environment_models/spherical_harmonics_gravity.cpp
double gravitationalParameter = ...
double referenceRadius = ...

Eigen::MatrixXd normalizedCosineCoefficients =  // NOTE: entry (i,j) denotes coefficient at degree i and order j
Eigen::MatrixXd normalizedSineCoefficients =  // NOTE: entry (i,j) denotes coefficient at degree i and order j

std::string associatedReferenceFrame = ...

bodySettings[ "Earth" ]->gravityFieldSettings = std::make_shared< SphericalHarmonicsGravityFieldSettings >(
   gravitationalParameter, referenceRadius, normalizedCosineCoefficients, normalizedSineCoefficients, associatedReferenceFrame );